sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -965,7 +965,11 @@ def set_lpmode(self, lpmode):
                 # Force module transition to LowPwr under SW control
                 lpmode_val = lpmode_val | (1 << CmisApi.LowPwrRequestSW)
                 self.xcvr_eeprom.write(consts.MODULE_LEVEL_CONTROL, lpmode_val)
-                time.sleep(0.1)
+                for retries in range(50):
```

Comment:
Could you please get the exact time taken by the module to get into LP mode?.

Reply:
We test set lpmode to on, need 2sec and can get status=True.
For lpmode=off, it need 3-4sec and can get get status=True. Otherwise , they will get false in current code.

Reply:
@jostar-yang please use [get_datapath_deinit_duration()](https://github.com/sonic-net/sonic-platform-common/blob/master/sonic_platform_base/sonic_xcvr/api/public/cmis.py#L690) to know how much time it take for module to transition to LowPwr mode i.e `ModuleDeactivatedT`

Reply:
Call get_datapath_init_duration until mstate == 'ModuleReady'. It get "60000".
root@sonic:~# sfputil lpmode off Ethernet248
Disabling low-power mode for port Ethernet248 ... 1:spend set_lpmdoe=off ,use 54.217353
BB:get dp_init_time=60000.000000

Reply:
@jostar-yang please update the code to read the timeout value and wait max duration for the advertised timeout